docs/source/simulators/SimulationPathFramework.rst

```diff
@@ -94,3 +94,8 @@ The framework can be configured using multiple options (which can be passed to t
  - :code:`seed`: the random seed used for the simulator (default :code:`0`, i.e., no particular seed)
 
 In addition to the above, CoTenGra can be configured using the options described [above](#cotengra).
+
+Usage as Standalone Executable
+##############################
+
+TODO
```

Comment:
I am not sure we ever exposed this functionality to the C++ executable. Maybe double check, I'd say this section can be removed for the moment or a warning can be added. The simulation path framework relies quite a bit on Python packages.

Reply:
Good point, I added the warning.